lib/isaac_configuration.go

```diff
@@ -1,8 +1,7 @@
-//
 // ISAACConfiguration has timeout features and transaction limit.
 // The ISAACConfiguration is included in ISAACStateManager and
 // these timeout features are used in ISAAC consensus.
-//
+
```

Comment:
Unrelated ?

Reply:
Yes 👍 but it is necessary.

Reply:
Why ?

Reply:
Ah, You want to remove new line, right?
I mean, removing empty comment.

Reply:
Well this style is used in many places. It allows to create a bit of visual space and not make the code too compact. I'm fine if you want to change that style, we can discuss it, but I just feel like it should be its own PR, since this file is untouched except for this comment change.